ckan-docker/ckan/Dockerfile.dev

```
@@ -1,58 +1,83 @@
+# Dockerfile.dev
 FROM ckan/ckan-base:ckan-2.10.0-dev
 
+# 1) Environment
+ENV APP_DIR=/srv/app \
```

Hadolint (2.14.0)

[error] Do not refer to an environment variable within the same `ENV` statement where it is defined.

(DL3044)